Base/usr/share/man/man7/boot_device_addressing.md

```diff
@@ -0,0 +1,61 @@
+## Name
+
+Boot Device Addressing - addressing the correct boot device to use.
+
+## Description
+
+Serenity's kernel can select boot device at boot time, based on the `root` boot parameter.
+This functionality is used to control which boot device is selected to be used for all further boot process operations.
+
+## Options
+
+The kernel `root` boot parameter take the form of **`root={value}`**, where the **`={value}`**
+trailer can be set to specific prefixes to indicate the boot device preference.
+
+### Addressing options
+
+The user can choose to use addressing based on synthetic unix concepts:
+
+```
+block0,0 [blockMAJOR:MINOR]
```

Comment:
The explanation in square brackets is confusing, especially compared to how it's done below.